src/style.css

```
@@ -1 +1,102 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf)
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir.ttc);
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf)
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Heavy.ttf);
}
```
And just specify font-weight